web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerDefault.java

```diff
@@ -0,0 +1,23 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+public class LinkCheckerDefault implements LinkCheckerInterface {
+    private String getCheckUrl(Element onlineResource) {
+        return LinkCheckerUtils.parseOnlineResource(onlineResource, LinkCheckerUtils.URL_XPATH);
+    }
+
+    @Override
+    public boolean check(Element onlineResource) {
+        String checkUrl = getCheckUrl(onlineResource);
+        if (checkUrl != null)
+            return LinkCheckerUtils.checkHttpUrl(checkUrl);
+
+        return true;
+    }
+
+    @Override
+    public String toString(Element onlineResource) {
```

Comment:
Yes, it's repeating. It happens to be that all those classes need a check url, but that's more of a coincidence. I can try extracting some of it to a base class, but what name will it have?